src/scripts/main.js

```diff
@@ -1,3 +1,391 @@
 'use strict';
+'use strict';
+
+const tbody = document.querySelector('tbody');
+const title = document.querySelector('thead');
+let revers = '';
```

Comment:
The variable 'revers' seems to be a typo. If you intended to use the word 'reverse', please correct it to avoid confusion.